packages/playwright-test/src/loader.ts

```diff
@@ -642,6 +642,8 @@ function validateProject(file: string, project: Project, title: string) {
   if ('use' in project && project.use !== undefined) {
     if (!project.use || typeof project.use !== 'object')
       throw errorWithFile(file, `${title}.use must be an object`);
+    if ((project.use as any).storageState !== undefined && (project.use as any).storageStateName !== undefined)
```

Comment:
We should not have this check in the `loader`, but rather in the fixtures themselves. However, I'd just prioritize one over the other and say that in the docs.